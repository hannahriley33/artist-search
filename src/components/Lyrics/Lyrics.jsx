import React from 'react';
import PropTypes from 'prop-types';

const Lyrics = ({ songTitle, artistName, songLyrics }) => {
  
  
  return (
    <section>
      <h4>{songTitle}</h4>
      <h5>{artistName}</h5>
      <h6>{songLyrics}</h6>
    </section>
  );
};

Lyrics.propTypes = {
  songTitle: PropTypes.string.isRequired,
  songLyrics: PropTypes.string.isRequired,
  artistName: PropTypes.string.isRequired
};

export default Lyrics;
