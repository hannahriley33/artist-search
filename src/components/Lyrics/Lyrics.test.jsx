import React from 'react';
import { shallow } from 'enzyme';
import Lyrics from './Lyrics.jsx';

describe('Lyrics', () => {
  it('displays song and its lyrics', () => {
    const wrapper = shallow(<Lyrics match={{ params: { artistId: 'singer', songTitle: 'my cool song, ' } }} />);
    expect(wrapper).toMatchSnapshot();
  });
});
