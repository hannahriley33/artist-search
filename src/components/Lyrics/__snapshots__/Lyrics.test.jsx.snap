// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Lyrics displays song and its lyrics 1`] = `
<section>
  <h4>
    is this the right format
  </h4>
  <h6>
    is this the right format
  </h6>
</section>
`;
